### PR TITLE
Add QBFT support for evmtool

### PR DESCRIPTION
**Pull request by usmansaleem**
## PR description
Support QBFT in evmtool. #2749 

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

File: ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/QBFTGenesisFileModule.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright ConsenSys AG.
```
Comment (jflo) [inline ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/QBFTGenesisFileModule.java:2]: New files to have updated copyright for Contributors to Hyperledger Besu

**Review by jflo — APPROVED** [review]

File: ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/QBFTGenesisFileModule.java (hunk 1 above)
Comment (usmansaleem) [reply ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/QBFTGenesisFileModule.java]: Thank you, updated.

File: ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/QBFTGenesisFileModule.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  QBFTGenesisFileModule(final String genesisConfig) {
+    super(genesisConfig);
+  }
+
+  @Override
+  ProtocolSchedule provideProtocolSchedule(
+      final GenesisConfigOptions configOptions,
+      @Named("RevertReasonEnabled") final boolean revertReasonEnabled) {
+    return QbftProtocolSchedule.create(
```
Comment (jframe) [inline ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/QBFTGenesisFileModule.java:39]: Should be able to use the simpler create constructor here, i.e. QbftProtocolSchedule.create(configOptions, bftExtraDataEncoder)

File: ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/QBFTGenesisFileModule.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  QBFTGenesisFileModule(final String genesisConfig) {
+    super(genesisConfig);
+    bftExtraDataEncoder = new QbftExtraDataCodec();
+  }
+
+  @Override
+  ProtocolSchedule provideProtocolSchedule(
+      final GenesisConfigOptions configOptions,
+      @Named("RevertReasonEnabled") final boolean revertReasonEnabled) {
+    return QbftProtocolSchedule.create(configOptions, revertReasonEnabled, bftExtraDataEncoder);
```
Comment (jframe) [inline ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/QBFTGenesisFileModule.java:38]: Don't think it's necessary to pass in the revertReasonEnabled as looks like this is handled by the EvmToolCommand itself

Comment (jframe) [reply ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/QBFTGenesisFileModule.java:38]: Actually think I'm wrong about this probably is needed

**Pull request merged**